.gitlab-ci.yml

```
@@ -888,6 +888,14 @@ library:ci-trakt:
   - plugin:ci-elpi_hb
   stage: build-2
 
+library:ci-wasm:
```

It seems to be a plugin not (just) a library so should be 
```suggestion
plugin:ci-wasm:
```

---

It is in fact a library of Wasm mechanisation in Coq -- maybe the proposed name was misleading?

---

I thought https://github.com/WasmCert/WasmCert-Coq/tree/master/src was a plugin but I guess not?